sites/friday/src/manage-resources/resource-init.md

```diff
@@ -106,79 +172,223 @@ platform integration:add \
+```
+
+For example, to use the `minimum` resource initialization strategy, run the following command:
+
+```bash {location="Terminal"}
+environment:branch --resources-init=minimum
+```
+
+<--->
+
++++
+title=In the Console
++++
+
+1. Navigate to the environment you want to branch from.
+2. Click {{< icon branch >}} **Branch**.
+3. Enter a name and a type for the new environment.
+4. Select an initialization strategy from the proposed list.
+5. Click **Branch**.
+
+{{< /codetabs >}}
 
 ### Environment merge
 
 When you [merge](/glossary.md#merge) a child environment into a parent environment,
 any app or service on the child environment is merged into the parent.
-For the merge and deployment to be complete,
-you need to [allocate resources](/manage-resources/adjust-resources.md) to each of these newly added instances.
+By default, the resources from the child environment are also merged into the parent.
+
+However, you can set another resource initialization strategy.
+
+{{% note theme="info" title="Prerequisite" %}}
+
+Make sure `sizing_api_enabled` is set to `true` in your `{{% vendor/configfile "app" %}}` file.
+
+{{% /note %}}
+
+{{< codetabs >}}
+
++++
+title=Using the CLI
++++
+
+Run the following command:
+
+```bash {location="Terminal"}
+environment:merge --resources-init={{< variable "INITIALIZATION_STRATEGY" >}}
+```
+
+For example, to use the `manual` resource initialization strategy, run the following command:
+
+```bash {location="Terminal"}
+environment:merge --resources-init=manual
+```
+
+<--->
+
++++
+title=In the Console
++++
+
+1. Navigate to the environment you want to merge into its parent.
+2. Click {{< icon merge >}} **Merge**.
+3. Select an initialization strategy from the proposed list.
+5. Click **Merge**.
+
+{{< /codetabs >}}
+
+### Environment activation
+
+When you activate an environment, {{% vendor/name %}} uses the same resource allocation as the parent environment.
+If there is no parent environment, the [`default` resource initialization strategy](#default-resources) applies.
+
+You can also set another resource initialization strategy using the CLI.
+
+{{% note theme="info" title="Prerequisite" %}}
+
+Make sure `sizing_api_enabled` is set to `true` in your `{{% vendor/configfile "app" %}}` file.
+
+{{% /note %}}
+
+Run the following command:
+
+```bash {location="Terminal"}
+environment:activate --resources-init={{< variable "INITIALIZATION_STRATEGY" >}}
+```
+
+For example, to use the `minimum` resource initialization strategy, run the following command:
+
+```bash {location="Terminal"}
+environment:activate --resources-init=minimum
+```
 
-### Environment sync
+### Backup restoration
+
+When you [restore a backup](/environments/restore.md),
+{{% vendor/name %}} uses the same resource allocation as the parent environment.
+If there is no parent environment, the [`default` resource initialization strategy](#default-resources) applies.
```

Comment:
@nicogommen Same question?

Reply:
At the moment, it is correct, yes.
But, we are maybe changing this... but it's not sure yet that it will make it for the release... 😅  
The change would be that we are saving the resources configured at the moment the backup is created. So when a backup is restored, we restore the environment with the exact resources as when the backup was taken.
This would be like a new strategy ("backup resources", or something like that), and it would be the default behaviour when restoring a backup.
Then you can still choose another strategy, as you described it.
So maybe we can put this section on hold for a short time. We should know very soon if this will make it in Git 26 or not.
I will let you know. 
Sorry about this change.

Reply:
Backup restoration section updated.